tests/aws/services/lambda_/test_lambda_api.snapshot.json

```diff
@@ -1,6 +1,6 @@
 {
   "tests/aws/services/lambda_/test_lambda_api.py::TestLambdaFunction::test_function_lifecycle": {
-    "recorded-date": "10-04-2024, 08:59:22",
+    "recorded-date": "12-09-2024, 11:29:18",
```

Comment:
Good to know (for next time ;): If nothing changes, we don't need to update the snapshots, but can run the test against AWS without `SNAPSHOT_UPDATE=1` to validate whether the snapshots still hold.

Again, no need to change anything (we also do that sometimes), just an extra info about the dev flow: AWS snapshot -> AWS validate -> LocalStack validate

Reply:
Thanks for the extra info, that’s really helpful! I’ll keep it in mind for the next time.